lib/sycamore/sycamore/transforms/extract_graph_entities.py

```diff
@@ -103,7 +104,7 @@ def _deserialize_entities(self):
         return create_model("entities", __base__=BaseModel, **fields)
 
     async def _extract_from_section(self, summary: str) -> str:
-        llm_kwargs = {"response_format": self._deserialize_entities()}
+        llm_kwargs = {"response_format": type_to_response_format_param(self._deserialize_entities())}
```

Comment:
I'm a little hesistent to use an openai function directly in the transform

Since this is openai specific formatting, is it better to let the openai.py file handle this?

Reply:
I agree, added the change + an additional try except loop for openai structured outputs. Sometimes the response gets cancelled because the LLM never reaches a stop token and repeats the same thing over and over again and in this scenario, I think the best failover is to return an empty dict back. It's bound to happen over large scale data and missing a document isn't the biggest deal.